content/en/containers/kubernetes/distributions.md

```diff
@@ -292,30 +294,11 @@ providers:
 {{% /tab %}}
 {{< /tabs >}}
 
-### Spot pods and instances
+### Spot pods and compute Classes
```

Comment:
```suggestion
### Spot pods and compute classes
```
I see "classes" is lower-case below, we should also lowercase here to adhere to sentence case for headings

Reply:
Oops I remembered to lower "compute" but forgot that.